unity/Assets/Editor/PointWaveWaterPlaneEditor.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEditor;
```

Comment:
Move this script into your experiment folder.
../scenes/experiments/PointWave/scripts/Editor/